src/core/icons/display/icon-display-something-else-mono.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="46" height="48" viewBox="0 0 46 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M21 2.1547C22.2376 1.44017 23.7624 1.44017 25 2.1547L40.9186 11.3453C42.1562 12.0598 42.9186 13.3803 42.9186 14.8094V33.1906C42.9186 34.6197 42.1562 35.9402 40.9186 36.6547L25 45.8453C23.7624 46.5598 22.2376 46.5598 21 45.8453L5.08142 36.6547C3.84381 35.9402 3.08142 34.6197 3.08142 33.1906V14.8094C3.08142 13.3803 3.84381 12.0598 5.08142 11.3453L21 2.1547Z" stroke="#03020D" stroke-width="1.5"/>
+<path d="M24.063 29.2105V27.4127C24.063 22.7202 29 23.5734 29 18.3629C29 15.0416 26.4882 13 23.3123 13C21.6089 13 19.8189 13.6704 18 14.4931V17.1136L23.1391 15.2548C25.3333 15.2548 26.6903 16.5651 26.6903 18.4543C26.6903 22.1413 21.6955 21.5319 21.9554 27.3213L22.042 29.2105H24.063ZM24.6982 35V31.6482H21.4646V35H24.6982Z" fill="#03020D"/>
```

Comment:
Mono icons are best when they can adapt to their surrounding colours - would suggest changing the fill/stroke properties to "currentColor" (take a look at other icons for inspiration). This doesn't need to be done for the colour version as its colours are opinionated